vnext/local-cli/src/runWindows/utils/deploy.ts

```diff
@@ -189,12 +224,21 @@ export async function deployToDevice(
 export async function deployToDesktop(
   options: RunWindowsOptions,
   verbose: boolean,
-  slnFile: string,
+  config: Config,
   buildTools: MSBuildTools,
 ) {
-  const appPackageFolder = getAppPackage(options);
+  const windowsConfig: WindowsProjectConfig = config.project.windows;
```

Comment:
As is, this type expresses windowsConfig to always be defined, and always have all properties. We later do check these cases, so I'm not sure this type expresses what we want it to. 

Looking at projectConfigWindows, the return type is  "WindowsProjectConfig | null". It seems like that's the right type here, and that we should be able to avoid null checking properties since WindowsProjectConfig doesn't allow them to be null or undefined. 

If we did want to express that windowsConfig can be undefined and that it's properties may be undefined, you could use a type like "Partial\<WindowsProjectConfig\> | undefined". I'm not sure that's correct though. 

Adding this extra information lets the typescript compiler enforce null/undefined checks correctly.

Reply:
thanks, will try this